packages/cli/src/commands/create/package-manager.ts

```diff
@@ -6,7 +8,27 @@ import { DEFAULT_VALUES, type CreateOptions } from './options.js';
 
 export type PackageManager = 'npm' | 'yarn' | 'pnpm' | 'bun';
 
-export async function selectPackageManager(flags: CreateOptions) {
+const LOCK_FILES = new Map<PackageManager, string>([
+  ['npm', 'package-lock.json'],
+  ['pnpm', 'pnpm-lock.yaml'],
+  ['yarn', 'yarn.lock'],
+]);
+
+export async function selectPackageManager(cwd: string, flags: CreateOptions) {
+  const packageManager = await resolvePackageManager(flags);
+
+  // remove lock files for other package managers
+  for (const [pkgManager, lockFile] of LOCK_FILES) {
+    if (pkgManager !== packageManager) {
+      console.log('Removing', path.join(cwd, lockFile));
```

Comment:
Oh this `console.log` should probably be removed. Or if we want to keep it then we should use `prompts.log.info`.

Reply:
Ugh, my own debug logs. Good catch.